public/less/d2h.less

```diff
@@ -55,6 +55,18 @@
   color: #000000;
 }
 
+.d2h-code-line,
+.d2h-code-side-line {
+  user-select: none;
+  width: 100%;
```

Comment:
Looks like this will make selecting text easier so you can start dragging the mouse anywhere in a line and it does not need to be within text right?

Reply:
Yeah -- it also requires this bit: https://github.com/simonwiles/ungit/blob/ef7543b466c902ddadf4f241da45bb32de28352e/public/less/d2h.less#L64-L68
```css
.d2h-code-line-ctn {
  display: inline-block;
  user-select: text;
  width: 100%;
}
```

so that the text-line container's parent has width too.